docs/reference/generators.rst

```diff
@@ -0,0 +1,61 @@
+.. _generators:
+
+===================================
+Generators and Application Modeling
+===================================
+
+.. currentmodule:: dimod.generators
+
+.. automodule:: dimod.generators
+
+Benchmarking
+============
+
+.. autosummary::
+   :toctree: generated/
+
+   anti_crossing_clique
+   anti_crossing_loops
+   chimera_anticluster
+   doped
+   frustrated_loop
+
+Constraints
+===========
```

Comment:
Consider having a category for Boolean Gates, which is useful for many problems

Reply:
Trying to strike a balance between the number of categories and the specificity. IMO we overestimate the value of the gate problems because they are used so much in some of our intro examples. I don't see them very much in any practical applications.